docs/resources/source_verification.md

```diff
@@ -80,9 +80,10 @@ Optional:
 - `synctera` (Attributes) (see [below for nested schema](#nestedatt--verification--synctera))
 - `tebex` (Attributes) (see [below for nested schema](#nestedatt--verification--tebex))
 - `telnyx` (Attributes) (see [below for nested schema](#nestedatt--verification--telnyx))
-- `threedeye` (Attributes) (see [below for nested schema](#nestedatt--verification--threedeye))
+- `three_d_eye` (Attributes) (see [below for nested schema](#nestedatt--verification--three_d_eye))
```

Comment:
breaking change, instead of "threedeye", the logic in the script uses "three_d_eye" instead. Let me know what you think how we should move forward here.

Reply:
That's fine